.github/workflows/ci.yaml

```diff
@@ -69,14 +72,36 @@ jobs:
         with:
           name: coverage-xml
       - name: Fix coverage file for sonarcloud
-        run: sed -i "s/home\/runner\/work\/starlite-jwt\/starlite_jwt/github\/workspace/g" coverage.xml
+        run: sed -i "s/home\/runner\/work\/starlite\/starlite/github\/workspace/g" coverage.xml
       - name: SonarCloud Scan
         uses: sonarsource/sonarcloud-github-action@master
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+  snyk:
+    needs:
+      - test
+    if: github.event.pull_request.head.repo.fork == false
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Run Snyk Monitor
+        if: ${{ github.ref == 'refs/heads/main' }}
+        uses: snyk/actions/python-3.8@master
+        with:
+          command: monitor
+        env:
+          SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
```

Comment:
we need to set this secret for this to work.

Reply:
ok, i'll leave it out for now and fix it in a followup PR